BeeInterns1/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Abel&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>BeeInterns</title>
+  </head>
+  <body class="page">
+    <header class="page_header header">
+      <div class="header_logo logo">&nbsp;</div>
+      <nav class="header_nav nav">
+        <div href="#" class="nav_item">&nbsp;</div>
+        <div href="#" class="nav_item">&nbsp;</div>
+        <div href="#" class="nav_item">&nbsp;</div>
+        <div href="#" class="nav_item">&nbsp;</div>
+      </nav>
```

Comment:
Думаю по заданию здесь должны быть ссылки вместо div, навигацию лучше оформлять списком 
```suggestion
      <nav class="header_nav nav">
         <ul class="nav_list">
            <li class="nav_item">
                <a href="#" class="nav_link"></a>
            </li>
           ...
        </ul>
     </nav>
```